README.md

```diff
@@ -46,6 +46,12 @@ The typical command for running the pipeline is as follows:
 ```
 nextflow run SciLifeLab/NGI-RNAseq --reads '*_R{1,2}.fastq.gz' --genome 'GRCm38'
 ```
+or using a more manual approach ( require you to clone the git repository)
+
+```
+nextflow path_to_NGI-RNAseq/main.nf -c path_to_NGI-RNAseq/example_uppmax_config --reads '*_R{1,2}.fastq.gz' --genome 'GRCm38'
```

Comment:
Nice, though maybe change `-c path_to_NGI-RNAseq/example_uppmax_config` to `-c path_to_your_nextflow_config`?

I can imagine people copying this command and then not understanding why the `example_uppmax_config` file doesn't work for them..